pyproject.toml

```diff
@@ -1,6 +1,79 @@
+[build-system]
+requires = [
+    "setuptools >= 40.8.0",
+    "wheel >= 0.29.0",
+]
+build-backend = 'setuptools.build_meta'
+
+[project]
+name = "galaxy-ng"
+description = "Galaxy NG plugin for the Pulp Project"
+readme = "README.md"
+license = {file = "LICENSE"}
+requires-python = ">=3.9"
+authors = [
+    { name = "Red Hat, Inc.", email = "info@ansible.com" },
+]
+keywords = ["ansible", "galaxy", "collection", "role"]
+classifiers = [
+    "Framework :: Django",
+    "License :: OSI Approved :: GNU General Public License v2 or later (GPLv2+)",
+    "Operating System :: POSIX :: Linux",
+    "Development Status :: 5 - Production/Stable",
+    "Framework :: Django",
+    "Programming Language :: Python",
+    "Programming Language :: Python :: 3",
+    "Programming Language :: Python :: 3.9",
+    "Programming Language :: Python :: 3.10",
+    "Programming Language :: Python :: 3.11",
+    "Programming Language :: Python :: 3.12",
+]
+dynamic = ["version"]
+dependencies = [
+    "boto3",
+    "distro",
+    "django-ansible-base[jwt-consumer,feature-flags] @ git+https://github.com/ansible/django-ansible-base@2025.3.7",
+    "django-auth-ldap==4.0.0",
+    "django-crum==0.7.9",
+    "django-flags>=5.0.13",
+    "django-ipware<4.0.0,>=3.0.0",
+    "django-picklefield<4.0.0,>=3.0.1",
+    "django-prometheus>=2.0.0",
+    "drf-spectacular",
+    "dynaconf>=3.2.10",
+    "galaxy-importer>=0.4.27,<0.5.0",
+    "insights_analytics_collector>=0.3.0",
+    "marshmallow<4.0.0,>=3.6.1",
+    "pulp-container>=2.19.2,<2.20.0",
+    "pulp_ansible==0.23.1",
+    "pulpcore>=3.49.0,<3.50.0",
+    "social-auth-app-django>=5.2.0",
+    "social-auth-core>=4.4.2",
+]
```

Comment:
we could do it dynamically  as:  https://github.com/ansible/aap-gateway/blob/devel/pyproject.toml#L27-L31